exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt

```diff
@@ -730,6 +730,9 @@ fun decimalParam(value: BigDecimal): Expression<BigDecimal> = QueryParameter(val
 /** Returns the specified [value] as a blob query parameter. */
 fun blobParam(value: ExposedBlob): Expression<ExposedBlob> = QueryParameter(value, BlobColumnType())
 
+/** Returns the specified [value] as an array query parameter. */
+fun <T> arrayParam(value: Array<T>): Expression<Array<T>> = QueryParameter(value, UntypedAndUnsizedArrayColumnType)
```

Comment:
Please remove this as it isn't necessary for the `Op` implementation. An `arrayParam()` will be introduced to the API when the `ArrayColumnType` is fully supported for storing arrays.